canonical-web-and-design/www.ubuntu.com
# /legal/data-privacy: add link to Third Party and Geolocation Policy

## Done

- Uploaded Third Party and Geolocation Policy pdf asset
- Linked to above asset from /legal/data-privacy

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/legal/data-privacy](http://0.0.0.0:8001/legal/data-privacy)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Ensure that the copy matches the [brief](https://docs.google.com/document/d/1HPe3ixFs_hC1pNxmfn11uI3S4c95tq2sJTFsO4AAzm0)


## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/1225
